theme-data/light/light-common.json

```diff
@@ -235,7 +235,7 @@
       "attention": "@color-red-60",
       "stable": "@color-green-60",
       "locked": "@color-gray-60",
-      "caution": "@color-yellow-50",
+      "caution": "@color-yellow-60",
```

Comment:
<img width="504" alt="Screenshot 2023-03-16 at 21 25 22" src="https://user-images.githubusercontent.com/128048536/225631464-eee0c3e7-1ef1-4918-8d3d-3c0dbabefb36.png">
https://www.figma.com/file/yYfvFEEr3UkW1yUKkW7zEH/Tokens---Theme-Color?node-id=861-6639&t=2WfGU7Dj2UkbSNCv-0
the indicator-caution color is yellow-60 on light mode.